client/.env.local

```diff
@@ -0,0 +1,4 @@
+This is a sample version of .env file. Provide values before running the app.
+
+REACT_APP_MAPBOX_TOKEN=pk.eyJ1IjoiZ3JleS1tYXR0ZXIiLCJhIjoiY2szN3p6YW1hMDRjYjNta3p3aW5waGVxOSJ9.ExrS6d3sQE1miLNZE9P-bw
```

Comment:
don't commit this file. should be added to .gitignore

Reply:
confused because the .env is already in the gitignore